README.md

```diff
@@ -246,16 +246,16 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
 
 | OpenTelemetry Operator | Kubernetes           | Cert-Manager         |
 |------------------------|----------------------|----------------------|
-| v0.41.0                | v1.20 to v1.22       | 1.6.1                |
-| v0.40.0                | v1.20 to v1.22       | 1.6.1                |
-| v0.39.0                | v1.20 to v1.22       | 1.6.1                |
-| v0.38.0                | v1.20 to v1.22       | 1.6.1                |
-| v0.37.1                | v1.20 to v1.22       | v1.4.0 to v1.6.1     |
-| v0.37.0                | v1.20 to v1.22       | v1.4.0 to v1.5.4     |
-| v0.36.0                | v1.20 to v1.22       | v1.4.0 to v1.5.4     |
-| v0.35.0                | v1.20 to v1.22       | v1.4.0 to v1.5.4     |
-| v0.34.0                | v1.20 to v1.22       | v1.4.0 to v1.5.4     |
-| v0.33.0                | v1.20 to v1.22       | v1.4.0 to v1.5.4     |
+| v0.41.0                | v1.21 to v1.23       | 1.6.1                |
```

Comment:
This is not correct. Released versions shouldn't be changed, the matrix is changed only from this point and on. So, just add a new entry like this:

```
| v0.42.0 (unreleased)               | v1.21 to v1.23       | 1.6.1                |
```

Reply:
Noted.

Reply:
@jpkrohling  Please check, I have updated

Reply:
0.41.0 has been released already without this PR, so, it really has to be "0.42.0 (unreleased)" here.

Reply:
@jpkrohling updated.

Reply:
0.41.0 is still wrong, it shouldn't be changed at all, as it has been released already.